helidon-ls-ext/helidon.jdt.ext/.gitignore

```diff
@@ -0,0 +1,88 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
`.gitignore` is recursive can you combine this with the one in the top `ls` folder or the root directory ?

Reply:
done